### PR TITLE
BE: Chore: standarize Protobuf import paths

### PR DESCRIPTION
**Pull request by yeikel**
**What changes did you make?** (Give an overview)

Before this change, the `SchemaRegistryServiceTests` tests were failing in Windows with the following exception: 

image

```java
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:782)
	... 55 more
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.kafbat.ui.service.DeserializationService]: Constructor threw exception
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:221)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:117)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:315)
	... 69 more
Caused by: java.lang.NullPointerException: ProtoFile not found for import 'language/language.proto'
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:1011)
	at io.kafbat.ui.serdes.builtin.ProtobufFileSerde$ProtoSchemaLoader$1.load(ProtobufFileSerde.java:403)
	at com.squareup.wire.schema.Linker.getFileLinker$wire_schema(Linker.kt:95)

```

With this change, we are standardizing the import statements to remove backlashes in Windows which fixes the tests : 

image

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

**A picture of a cute animal (not mandatory but encouraged)**
istockphoto-1243410279-612x612

### CODE REVIEW

File: api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java [hunk 1]
```diff
@@ -426,6 +427,27 @@ private Map<String, ProtoFile> loadFilesWithLocations() {
       }
       return filesByLocations;
     }
+
+    /**
```
Comment (Haarolean) [inline api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java:431]: Do we keep this until the issue is resolved on wire side? Or is this a different issue?

Comment (yeikel) [reply api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java:431]: I am not sure what issue you are referring to

I noticed this problem locally when I ran these tests manually

Comment (Haarolean) [reply api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java:431]: Sorry I thought we've discussed this with you. See #711 and https://github.com/kafbat/kafka-ui/pull/712#discussion_r1885852375

This approach is more clean and I don't mind accepting it as a temporarily solution until the bug is fixed in wire.

Comment (yeikel) [reply api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java:431]: Ah, I see. Thanks for the context

When you say "bug in wire," does that refer to an eternal bug report you can point me to? Or an internal module that needs attention?

Either way, this temporary fix would be nice to allow running under Windows

Comment (Haarolean) [reply api/src/main/java/io/kafbat/ui/serdes/builtin/ProtobufFileSerde.java:431]: Wire bug relates to this: https://github.com/kafbat/kafka-ui/pull/712#issuecomment-2543916366 -> https://github.com/square/wire/pull/3215

**Review by Haarolean — APPROVED** [review]

**Pull request merged**